docs/tools/lsp6-signerjs/getting-started.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1.1
```

Comment:
no need to have float here - only integers.

There was this thing that some people started using float numbers and then every one started putting floats everywhere. It is not needed.

```suggestion
sidebar_position: 1
```